src/vmspawn/vmspawn.c

```diff
@@ -679,6 +679,15 @@ static int start_tpm(
         if (!ssp.listen_address)
                 return log_oom();
 
+        _cleanup_free_ char *swtpm_setup = NULL;
+        r = find_executable("swtpm_setup", &swtpm_setup);
```

Comment:
Does this leave state around? How is it handled?

Reply:
RuntimeDirectory= cleans it up. this is not changed by this patc.

swtpm already writes to the dir, this here just sets it up explicitly with some config, instead of doing so implicitly with the default config that swtpm would set up if you call it without anything.

Reply:
ok, I had missed it was using RuntimeDir